core/src/main/java/org/stellar/anchor/sep45/ISep45Service.java

```diff
@@ -0,0 +1,14 @@
+package org.stellar.anchor.sep45;
+
+import org.stellar.anchor.api.exception.AnchorException;
+import org.stellar.anchor.api.sep.sep45.ChallengeRequest;
+import org.stellar.anchor.api.sep.sep45.ChallengeResponse;
+import org.stellar.anchor.api.sep.sep45.ValidationRequest;
+import org.stellar.anchor.api.sep.sep45.ValidationResponse;
+
+public interface ISep45Service {
```

Comment:
```suggestion
public interface Sep45Service {
```
Unless `ISep45Service` is used somewhere else, we don't have to create the interface.

For `ISep10Service`, it was probably created for some legacy reasons that I can't remember. May be we may remove it as well.